test/base.py

```diff
@@ -0,0 +1,93 @@
+import yaml
+
+from api.api import get_album_by_id, get_playlist_by_id, create_playlist, add_items_to_playlist, update_playlist_details, delete_unfollow_a_playlist, get_current_user_profile, delete_items_from_playlist, get_playlist_items
```

Comment:
Please format this import part: separate the imported items with newline between brackets